de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/stringtable/FileTable.java

```diff
@@ -6,13 +6,17 @@
 	public static final String PROJECT_EXPLORER_VIEW_FXML = Paths.get("ui", "view", "ProjectExplorerView.fxml")
 			.toString();
 
+	public static final String NEW_FILE_PAGE_VIEW = Paths.get("ui", "view", "NewFileView.fxml").toString();
```

Comment:
Some nitpicking here: maybe a constant for ui/view?

Reply:
makes sense